README.md

```diff
@@ -1,25 +1,91 @@
 
-The [Demonstration Chain](https://psibase.io/) hosts both the [About Page](https://about.psibase.io/) and the [Documentation](https://doc-sys.psibase.io/).
+| Package   | Description                                                                   |
+|-----------|-------------------------------------------------------------------------------|
+| `psinode` | The psibase infrastructure node                                               |
+| `psibase` | Command-line utility for interacting with psinode. Used primarily by scripts. |
+| `psidk`   | The SDK used for developing web services                                      |
 
-## Binary Release
+-------
 
-If you want to operate a node, build services, or explore psibase's command-line utility, we recommend installing a [Binary Release](https://github.com/gofractally/psibase/releases). We provide a single binary package which should run on most recent Linux distributions. See the [installation instructions](https://doc-sys.psibase.io/linux.html).
+<p align="center">
+    &bull;
+    <a href="#demo-deployment-%EF%B8%8F">Demo 🖥️</a> &bull;
+    <a href="#technical-docs-">Technical docs 📓</a> &bull;
+    <a href="#running-psibase-">Running psibase 🚀</a> &bull;
+    <a href="#build-from-source-%EF%B8%8F">Build from source 🛠️</a> &bull;
+    <a href="#contribute-">Contribute 🧬</a> &bull;
+    <a href="#support-">Support 📱</a> &bull;
+</p>
 
-## Building this repo from source
+-------
 
-See the architecture-specific instructions below.
+## Demo deployment 🖥️
 
-Set the `WASI_SDK_PREFIX` environment variable before building. Alternatively, use cmake's `-DWASI_SDK_PREFIX=....` option. Also make sure `node 16.17`, `npm 8.19`, `yarn 1.22`, `cargo 1.63`, and `wasm-opt` (or later versions) are in your path.
+A demonstration deployment is currently hosted at [psibase.io](https://psibase.io) to showcase what psibase infrastructure could look like for end users. The applications available on this demo are sample applications and are not intended to be used for a production deployment.
+
+## Technical docs 📓
+
+Documentation is hosted on the [doc-sys applet](https://doc-sys.psibase.io/) of our demo deployment.
+
+## Running psibase 🚀
+
+If you want to operate a node, build services, or explore psibase's command-line utility, we recommend one of the following methods:
+1. Installing a binary release, the latest of which can be found on our [Releases](https://github.com/gofractally/psibase/releases) page. We provide binary packages for both Ubuntu 20.04 and Ubuntu 22.04.
+2. Installing our psinode [Docker Image](https://github.com/gofractally/psinode-docker-image/pkgs/container/psinode). Our docker image can be used to run psibase on any operating system that supports Docker. Currently we do not automatically build an image for the arm architecture. However, we have successfully manually built and run Psibase on the latest [Apple Silicon](https://support.apple.com/en-us/HT211814) based devices. 
```

Comment:
I think the discussion of arm is too focused on our processes rather than what it means for users. Users will not care whether it's automatic or manual for us. They only care whether we provide a pre-built image or they need to build it themselves.